doc/user.rst

```diff
@@ -65,35 +65,14 @@ matrix generators ``randsym`` and ``randorth``::
   """
   randorth(n) = Matrix(qr(randn(n,n)).Q)
 
-We first need to find out where the user directory of Matrix Depot is installed. This 
-can be done by::
+We can then write::
 
-    julia> MatrixDepot.user_dir()
-    "/home/.../.julia/dev/MatrixDepot/myMatrixDepot"
+  MatrixDepot.include_generator(FunctionName, "randsym", randsym)
+  MatrixDepot.include_generator(FunctionName, "randorth", randorth)
 
-That points to the default user directory which can be changed by an environment variable::
+and when we are done including generators, we need to reinitializie the database::
 
-    MATRIXDEPOT_USERDIR=/...
-
-The data directory is queried by::
-
-    julia> MatrixDepot.data_dir()
-    "/home/.../.julia/scratchspaces/b51810bb-c9f3-55da-ae3c-350fc1fbce05/data"
-
-By default, the data directory is managed by ``Scratch.jl``, but can be changed by another environment variable::
-
-    MATRIXDEPOT_DATA=/...
-
-For me, the package user data are installed at
-``/home/.../.julia/dev/MatrixDepot/myMatrixDepot``. We can copy ``myrand.jl`` to this directory.
-Now we open the file
-``myMatrixDepot/generator.jl`` and write::
-
-  include_generator(FunctionName, "randsym", randsym)
-  include_generator(FunctionName, "randorth", randorth)
-
-The changes are activated by re-initializing::
-    julia> MatrixDepot.init()
+  MatrixDepot.init()
```

Comment:
May solvable by `markdown.jl:29`
```
function mdinfo(data::GeneratedMatrixData)
    md = Docs.doc(data.func)
```
instead of
```
function mdinfo(data::GeneratedMatrixData)
    md = eval(Meta.parse("Docs.@doc $(data.func)", raise = false))
```

Reply:
resolved in https://github.com/JuliaMatrices/MatrixDepot.jl/pull/79/commits/b29c3d85cd058f0296a35ac5e4c5ac8a6172cc46